src/Calculator/Views/DateCalculator.xaml.cpp

```diff
@@ -186,7 +186,27 @@ void DateCalculator::SetDefaultFocus()
 void DateCalculator::DateCalcOption_Changed(_In_ Platform::Object ^ sender, _In_ Windows::UI::Xaml::Controls::SelectionChangedEventArgs ^ e)
 {
     FindName("AddSubtractDateGrid");
-    DateCalculationOption->SelectionChanged -= m_dateCalcOptionChangedEventToken;
```

Comment:
Do we still need to unregister from the event?

Reply:
I don't think so, since the event is going to continually fire on selection changed and we want the listener to respond and swap back and forth the date, it makes sense to leave it. I think before my change, it was viewed as a lazy load one time initialize type of function, but now there's more logic that has to execute.